zerver/worker/queue_processors.py

```diff
@@ -480,6 +482,17 @@ def get_bot_api_client(self, user_profile):
 
     def get_bot_handler(self, service):
         # type: (Service) -> Any
+
+        # Convert unicode service.name to string.
+        str_service_name = unicodedata.normalize('NFKD', service.name).encode('ascii', 'ignore')
```

Comment:
Why do you have this unicode normalization logic?  We probably should just enforce that `service_name` values are ascii... if we don't, I'd like to change that in the code paths that create `Service` objects in its own commit (with tests!).